@navikt/core/react/src/alert/base-alert/root/BaseAlertRoot.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { forwardRef } from "react";
+import { useRenameCSS } from "../../../theme/Theme";
+import { AkselColor } from "../../../types";
+import {
+  type BaseAlertContextProps,
+  BaseAlertProvider,
+} from "./BaseAlertRoot.context";
+import { baseAlertVariantToDataColor } from "./BaseAlertRoot.utils";
+
+interface BaseAlertProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Component content.
+   */
+  children: React.ReactNode;
+  /**
+   * Changes the size of the BaseAlert.
```

Comment:
Since this will be inherited by other components, maybe we shouldn't mention "BaseAlert"?
```suggestion
   * Changes the size.
```